cpp/include/cudf/strings/find.hpp

```diff
@@ -103,6 +104,28 @@ std::unique_ptr<column> contains(
   string_scalar const& target,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Returns a column of boolean values for each string where true indicates
+ * the corresponding target string was found within that string in the provided column.
+ *
+ * The 'output[i] = true` if string `targets[i]` is found inside `strings[i]` otherwise
+ * `output[i] = false`.
+ * If `target[i]` is an empty string, true is returned for `output[i]`.
+ *
+ * Any null string entries return corresponding null entries in the output column.
```

Comment:
Does `entries` include `targets[i]` as well as `strings[i]`?

Reply:
No. Just `strings[i]`. Null `targets[i]` will set the `output[i]` to false.

Reply:
Updated doxygen with this clarification.